### ISSUE TITLE
Need ability to write to buffer with full colour

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mabe02/lanterna#93]
```
I'm creating a roguelike. I need full colour support, and buffer support; I 
noticed that I can get full colour via Terminal, and buffer via Screen, but not 
both, simultaneously.

Right now, I try to draw the absolute minimum changed characters, because I get 
a LOT of drawing artefacts. 

Here are some proposals:

- New Screen.putString method with RGB foreground colour (eg. [255, 128, 0])
- Expose Screen.putChar (as protected), and extend ScreenCharacter to support 
RGB
- Expose Screen.backBuffer (as protected), and extend ScreenChar to support RGB

I don't know what else will work. Any of these will suffice for me.
```

Original issue reported on code.google.com by `alibhai....@gmail.com` on 28 Nov 2013 at 5:14

**Comment by mabe02** [comment]
Quick update; Lanterna 3.0-beta1 was released to Maven Central a while back and it's becoming more and more useful. It has full color support in the Screen layer so please check it out if you have a chance.

**Comment by ashes999** [comment]
Noted, thanks. My Java rougelike is long dead, but if I get a chance, I'll at least verify that this issue is closed in 3.0-beta1.
